.github/workflows/analysis-psalm.yml

```diff
@@ -26,6 +26,8 @@ jobs:
             path: "./service-admin"
           - name: service-pdf
             path: "./service-pdf"
+          - name: shared
```

Comment:
You're probably going to need an autoloader here, and possibly a composer.json file, as this has dependencies of its own (if you're psalm scanning it).